Sources/UIComponents/Components/Buttons/ButtonConfigurations/ButtonConfiguration.swift

```diff
@@ -0,0 +1,86 @@
+import UIKit
+
+extension GOVUKButton {
+    public struct ButtonConfiguration {
+        let titleNormal: UIColor?
```

Comment:
Naming is hard :)

Let's have a chat about how to name these. I think we should consider them all together. It would seem to make sense to have a hierarchy in the name, so start with what it is eg `title`, then what property and then what state. So of the options you've suggested `titleColorNormal` would seem the best.